source/_lovelace/entities.markdown

```diff
@@ -222,6 +222,42 @@ hide_if_unavailable:
   default: false
 {% endconfiguration %}
 
+### Conditional
+
+Special row that displays based on entity states.
+
+{% configuration %}
+type:
+  required: true
+  description: conditional
+  type: string
+conditions:
+  required: true
+  description: List of entity IDs and matching states.
+  type: list
+  keys:
+    entity:
+      required: true
+      description: HA entity ID.
+      type: string
+    state:
+      required: false
+      description: Entity state is equal to this value.*
```

Comment:
You can't use a ⭐️ to indicate that something is required, Markdown doesn't like that.

Reply:
The `*` just highlights the additional information

Reply:
Ah, were you referring to @bramkragten suggestion to move the *, as that did indeed break the parser?